app/controllers/application_controller.rb

```diff
@@ -68,14 +68,12 @@ def store_location
 
   def after_sign_in_path_for(resource)
     if current_user.confirmed_at == nil
```

Comment:
nit. Can we use `if !current_user.confirmed_at` or `unless current_user.confirmed_at`?